README.md

```diff
@@ -58,7 +58,7 @@ In this newly created folder, change the file `default.json` to the following co
     },
     "miner": {
         "new_transactions_threshold": 8,
-        "type_hash": "0xaeeb7a04b3766acbf86b51b03aa97f4c7a409a1211ac259dc58bc7ad89805561",
+        "type_hash": "0x565cd9ac9a251109ff642090b91af29bfaa451b2d0e3093a29bba74743408a4a",
```

Comment:
Just curious, will we have a deterministic algorithm in the future that the PK will always produces a fixed(unchanged) type hash?

Reply:
There might be such an algorithm with respect to a certain SDK, but I don't think we will have one algorithm that works across CKB, since CKB's programming model is quite flexible, for example, CKB even doesn't have the concept of PK, PK only exists when we reason in the scope of this SDK.

Reply:
I see. That makes good sense.